src/main/java/com/moddy/server/service/offer/HairServiceOfferRetrieveService.java

```diff
@@ -36,4 +47,38 @@ public OpenChatResponse getOpenChatInfo(final Long userId, final Long offerId) {
 
         return openChatResponse;
     }
+
+    public DetailOfferResponse getOfferDetail(final Long modelId, final Long offerId) {
+
+        HairServiceOffer hairServiceOffer = hairServiceOfferJpaRepository.findById(offerId).orElseThrow(() -> new NotFoundException(ErrorCode.NOT_FOUND_OFFER_EXCEPTION));
+
+        DesignerInfoResponse designerInfoResponse = designerRetrieveService.getOfferDesignerInfoResponse(hairServiceOffer.getDesigner().getId());
+        StyleDetailResponse styleDetailResponse = getStyleDetailResponse(hairServiceOffer, offerId);
+        handleOfferClickStatus(hairServiceOffer);
+
+        return new DetailOfferResponse(designerInfoResponse, styleDetailResponse);
+    }
+
+    private StyleDetailResponse getStyleDetailResponse(final HairServiceOffer hairServiceOffer, final Long offerId) {
+
+        String applicationHairDetail = hairModelApplicationRetrieveService.fetchApplicationHairDetail(hairServiceOffer.getHairModelApplication().getId());
+        List<String> preferHairStyleList = hairModelApplicationRetrieveService.fetchPreferHairStyle(hairServiceOffer.getHairModelApplication().getId());
+
+        List<PreferOfferCondition> preferOfferConditionList = preferOfferConditionJpaRepository.findAllByHairServiceOfferId(offerId);
+        List<OfferCondition> offerConditionList = preferOfferConditionList.stream().map(PreferOfferCondition::getOfferCondition).collect(Collectors.toList());
+        List<Boolean> preferOfferConditionBooleanList = Arrays.stream(OfferCondition.values()).map(condition -> {
+            if (offerConditionList.contains(condition)) return true;
+            else return false;
+        }).collect(Collectors.toList());
```

Comment:
```suggestion
        List<Boolean> preferOfferConditionBooleanList = Arrays.stream(OfferCondition.values()).map(offerConditionList::contains).collect(Collectors.toList());
```
p2
contains 자체가 boolean을 반환하기 때문에 함다를 사용하여 위 코드 처럼 코드를 줄일 수 있을 것 같습니다